lib/controller.js

```diff
@@ -497,6 +510,7 @@ controller.provisionTessel = function(opts) {
 };
 
 controller.deployScript = function(opts) {
+  opts.authorized = true;
```

Comment:
I did a test earlier where I put it in put it in standardTesselCommand, but there's one place where it's a `false`, and I had other stuff to do so I didn't press it